rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -100,11 +100,14 @@ def test_ppo_compilation_and_schedule_mixins(self):
 
         num_iterations = 2
 
-        for fw in framework_iterator(config, frameworks=("torch", "tf2")):
+        for fw in framework_iterator(config, frameworks=("tf2")):
```

Comment:
Oh, I guess, this is all local testing code. Never mind.

Reply:
Yes :)